README.md

```diff
@@ -237,6 +237,7 @@ vous trouverez sur cette plateform des cours et examens qui vous accompagnerons
   * [RubyMine](https://www.jetbrains.com/ruby/) (Ruby)
   * [DataGrip](https://www.jetbrains.com/datagrip/) (Databases)
 * [CodeBlocks](http://codeblocks.org/)
+* [VisualStudioCode](https://code.visualstudio.com/download)
```

Comment:
Hi @sattwik21, thank you for your contribution. Unfortunately we already added it in [Text editors](https://github.com/open-minds/awesome-openminds-team#text-editors)